segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -40,13 +42,20 @@
     protected static final int APPENDS_PER_SEGMENT = 10;
     private static final int SEGMENT_COUNT = 4;
 
+    protected StorageMetricsBase metrics = null;
+
     @Override
     protected int getThreadPoolSize() {
         return 5;
     }
 
     //endregion
 
+    static {
```

Comment:
This is costly and causes GC errors. Ideally metrics should have a shutdown method. But as it does not have it, different instances keep consuming memory.